galaxy_ng/app/api/ui/urls.py

```diff
@@ -22,5 +23,5 @@
     path('', include(router.urls)),
     # NOTE: Using path instead of SimpleRouter because SimpleRouter expects retrieve
     # to look up values with an ID
-    path('me/', viewsets.CurrentUserViewSet.as_view({'get': 'retrieve'}))
+    path('me/', viewsets.CurrentUserViewSet.as_view({'get': 'retrieve', 'put': 'update'}))
```

Comment:
Same question here. I'm guessing we don't want the `put` method to be accessible when on c.rh.com.